interface.go

```diff
@@ -16,7 +16,7 @@ type ProcessConfig struct {
 	CreateStdInPipe  bool
 	CreateStdOutPipe bool
 	CreateStdErrPipe bool
-	ConsoleSize      [2]int
+	ConsoleSize      [2]uint
```

Comment:
Ah, re TP5: https://github.com/docker/docker/pull/26613 :smile:.  We do use it though to set the initial console size. Following updates are sent through resize.